shared/aggregation/attestations/maxcover.go

```diff
@@ -171,7 +144,7 @@ func (al attList) filterContained() attList {
 	sort.Slice(al, func(i, j int) bool {
 		return al[i].AggregationBits.Count() > al[j].AggregationBits.Count()
 	})
-	filtered := make([]*ethpb.Attestation, 0, len(al))
+	filtered := al[:0]
```

Comment:
Just relying on fact that `al[:0]` will be sharing the same underlying array, so no need to allocate anything - we are ok wit in-place amendment (see https://github.com/golang/go/wiki/SliceTricks#filtering-without-allocating)